### PR TITLE
typst: add page

### PR DESCRIPTION
**Pull request by mintyfrankie**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known): typst v0.6.0

### CODE REVIEW

File: pages/common/typst.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# typst
+
+> Compile a typst file to PDF.
+> More information: <https://github.com/typst/typst>.
+
+- List all discoverable fonts in the system and the given directory:
+
+`typst --font-path {{path/to/fontDir}} fonts`
```
Comment (kbdharun) [inline pages/common/typst.md:8]:
```suggestion
`typst --font-path {{path/to/fonts_directory}} fonts`
```

File: pages/common/typst.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# typst
+
+> Compile a typst file to PDF.
```
Comment (kbdharun) [inline pages/common/typst.md:3]:
```suggestion
> Compile a Typst file to PDF.
> Note: Specifying output location is optional.
```

File: pages/common/typst.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+> More information: <https://github.com/typst/typst>.
+
+- List all discoverable fonts in the system and the given directory:
+
+`typst --font-path {{path/to/fontDir}} fonts`
+
+- Compile a Typst file directly:
+
+`typst compile {{path/to/source.typ path/to/output.pdf}}`
```
Comment (kbdharun) [inline pages/common/typst.md:12]:
```suggestion
- Compile a Typst file directly:

`typst compile {{path/to/source.typ}} {{path/to/output.pdf}}`
```

File: pages/common/typst.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+- Watch changes on a Typst file and compile dynamically:
+
+`typst watch {{path/to/source.typ path/to/output.pdf}}`
```
Comment (kbdharun) [inline pages/common/typst.md:16]:
```suggestion
`typst watch {{path/to/source.typ}} {{path/to/output.pdf}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Comment by mintyfrankie** [comment]
Hi @kbdharun thanks for the review! LGTM and a new commit is submitted.

**Review by kbdharun — APPROVED** [review]

File: pages/common/typst.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# typst
+
+> Compile a Typst file to PDF.
+> Note: Specifying output location is optional.
```
Comment (acuteenvy) [inline pages/common/typst.md:4]:
```suggestion
> Note: Specifying the output location is optional.
```

File: pages/common/typst.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+- List all discoverable fonts in the system and the given directory:
+
+`typst --font-path {{path/to/fonts_directory}} fonts`
+
+- Compile a Typst file directly:
+
+`typst compile {{path/to/source.typ}} {{path/to/output.pdf}}`
+
+- Watch changes on a Typst file and compile dynamically:
```
Comment (acuteenvy) [inline pages/common/typst.md:15]:
```suggestion
- Watch a Typst file and recompile on changes:
```

File: pages/common/typst.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+
+> Compile a Typst file to PDF.
+> Note: Specifying output location is optional.
+> More information: <https://github.com/typst/typst>.
+
+- List all discoverable fonts in the system and the given directory:
+
+`typst --font-path {{path/to/fonts_directory}} fonts`
+
+- Compile a Typst file directly:
```
Comment (acuteenvy) [inline pages/common/typst.md:11]:
```suggestion
- Compile a Typst file:
```

**Comment by mintyfrankie** [comment]
Hi @acuteenvy suggestions received and accepted with thanks!

**Review by kbdharun — APPROVED** [review]

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**